x/profiles/keeper/msg_server_dtag_transfers.go

```diff
@@ -105,7 +105,7 @@ func (k MsgServer) AcceptDTagTransferRequest(goCtx context.Context, msg *types.M
 	}
 
 	// Get the current owner profile
-	currentOwnerProfile, exist, err := k.GetProfile(ctx, msg.Receiver)
```

Comment:
renaming `currentOwnerProfile` into `requestReceiverProfile` makes it more explicit that focus on receiver of the transfer request.